schwab.py

```diff
@@ -0,0 +1,174 @@
+#!/usr/bin/python
+#
+# Copyright 2021 Chetan Narsude <chetan+csv2txf@celeral.com>
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Implements Schwab
+
+Charles Schwab gain/loss output provides already-reconciled transactions, i.e.,
+each buy/sell pair comes in a single record, on a single line.
+
+Does not handle:
+* dividends
+* short sales
+* partial lot sales
+"""
+
+import csv
+from datetime import datetime
+from decimal import Decimal
+import re
+import utils
+
+
+FIRST_LINE = 'Description of property (Example 100 sh. XYZ Co.),Date acquired,Date sold or disposed,Proceeds,Cost or other basis,Accrued market discount,Wash sale loss disallowed,Short-Term gain loss Long-term gain or loss Ordinary,Form 8949 Code,Check if proceeds from collectibles QOF,Federal income tax withheld,Check if noncovered security,Reported to IRS: Gross proceeds Net proceeds,Check if loss is not allowed based on amount in 1d,Profit or (loss) realized in 2020 on closed contracts,Unrealized profit or (loss) on open contracts-12/31/2019,Unrealized profit or (loss) on open contracts-12/31/2020,Aggregate profit or (loss) on contracts,Check if basis reported to IRS,Bartering,State name,State identification no,State Tax Withheld\n'
```

Comment:
For ease of readability, would you mind splitting these into a list of fields, and keeping them in a list, and joining them via `','.join(['Description of property ...', 'Date acquired', ... ]) + '\n'`?